code/client/munkilib/fetch.py

```diff
@@ -70,14 +70,13 @@
     munki_version, darwin_version)
 
 
-def import_middleware():
-    '''Check munki folder for a python file that starts with 'middleware'.
+def import_middleware(file_prefix='middleware', globals_prefix='middleware', required_function_name='process_request_options'):
```

Comment:
Run your changes through a PEP-8 checker; this line is definitely over 80 characters...